docs/src/trouble/index.md

```diff
@@ -0,0 +1,76 @@
+# {#ref-ts} Troubleshooting
+
+If you receive one of the following errors: 
+
+```cmd
+reach: /app/tmp/out.sh: openFile: permission denied (Permission denied)
+```
+
+OR
+
+```
+chmod: cannot access '/tmp/reach.../out.sh': No such file or directory
+./reach: 1: /tmp/reach.../out.sh: not found
+```
+
+* This generally means that Reach was previously run or installed with `sudo` and directories were created that only `root` can access. 
+
+* If this is the case, delete the local copy of the Reach program and reinstall per the instructions.
+
+* If use of `sudo` during installation is not the cause then try [updating Docker](https://techdirectarchive.com/2021/10/17/how-to-manually-update-docker-desktop/) and Docker-compose.
+
+"I'm running Windows 8 and cannot install Docker."
+
+Unfortunately, Docker Desktop does not support Windows 8.
+Also, Reach is not able to run on Windows 8. 
+
+Possible solutions are:
+* Upgrade to Windows 10 or Windows 11.
+* Partition the drive (or use a second drive) and install of a copy of a supported Linux distro.
+* Run a VM with a supported OS.
+It is likely that this option has the worst experience due to the splitting of system resources.
+
+Executing `./reach version` on Linux outputs `./reach: Is a directory`
+
+* This error usually occurs when `reach` is being called from a directory where it is not installed.
+Compare where Reach is installed with the current working directory. 
+* Check the current working directory with the terminal command `pwd`.
+
+If running `./reach run` requires `sudo`, then it is likely that:
+
+* A component used by `reach` was installed/created using `sudo`, such as Docker or the folder directory.
```

Comment:
I disagree, because if you do `sudo mkdir`, you are creating a folder directory but you are not installing anything. I thought this would cause accessibility issues. Is that incorrect?

Reply:
Oh, I gotcha. I thought you were saying the whole thing was redundant.

Reply:
Docker also _has_ to be installed by a user with elevated privileges, so I think this line is likely to just confuse people.